highfidelity/hifi
# Add "Your Last Location" to Android Home

Changes in Android init behaviour
+ Not connecting to any domain at startup
+ New option "Your Last Domain" in the Home screen
+ We avoid audio issues at startup

Test plan:
- Uninstall interface from your phone before downloading this PR.
- Open Interface.  You should see "Your Last Location" at the top of the list of domains.  Go to any domain and have someone else go there (on desktop is fine).  Verify that they hear you when you speak.
- Close Interface (the other person remains in that domain).  Open Interface.  The other person shouldn't see you appear, or be able to hear you.
- Press "Your Last Location".  You should appear next to the other person, and they should be able to hear you.
- Domain images should appear correctly, and there should be a new image for domains with the default image.
